tests/acceptance/purchase-data/PurchaseDataCest.php

```diff
@@ -802,11 +802,8 @@ public function testSendPurchaseDataNameFormatHonoredWhenSubscribed(AcceptanceTe
 			]
 		);
 
-		// Confirm that the email address was now added to ConvertKit.
-		$I->apiCheckSubscriberExists($I, $result['email_address']);
-
-		// Confirm that the subscriber's name = First Last.
-		$I->apiCheckSubscriberEmailAndNameExists($I, $result['email_address'], 'First Last');
+		// Confirm that the email address was now added to ConvertKit with a valid name.
+		$subscriber = $I->apiCheckSubscriberExists($I, $result['email_address'], 'First Last');
```

Comment:
There were quite a few places where we had two calls to the API to confirm if a subscriber was added.  This isn't necessary, because `apiCheckSubscriberExists` supports optionally checking for the name format, making the second call to `apiCheckSubscriberEmailAndNameExists` redundant.

This probably should have been submitted as a separate PR, but I only spotted it whilst working through the test for the v4 API.